packages/cascara/src/modules/ActionButton/ActionButton.js

```diff
@@ -36,9 +36,9 @@ const ActionButton = ({
   // FDS-137: use action name for button name if no content is specified
   const buttonText = content || name;
 
-  const handleClick = ({ currentTarget }) => {
+  function handleClick({ currentTarget }) {
     onAction(currentTarget, record);
-  };
+  }
```

Comment:
Should be an arrow func

Reply:
![image](https://user-images.githubusercontent.com/13924868/110033649-48834800-7cff-11eb-9c90-4e7e537a07d1.png)

Reply:
we need to decide if we remove that rule or use arrow functions, we can't do both.

Reply:
We can keep the rule and use arrow functions: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md#react-hooks

Reply:
changed, thanks